src/server/middleware/validation/geoSchemes.ts

```diff
@@ -0,0 +1,35 @@
+import { query } from 'express-validator'
+
+import { ForestSource, precalForestAgreementSources } from '@meta/geo'
+
+export const forestLayerSchema = [
+  query('countryIso')
+    .exists()
+    .withMessage('Country ISO is required')
+    .isLength({ min: 3, max: 3 })
+    .withMessage('Country ISO must be a 3 char value')
+    .bail(),
+
+  query('forestSource')
+    .exists()
+    .withMessage('Forest source is required')
+    .isIn(precalForestAgreementSources)
+    .withMessage('Not valid forest source'),
+
+  query('gteHansenTreeCoverPerc')
+    .if(query('forestSource').equals(ForestSource.Hansen))
+    .exists()
+    .withMessage('Hansen tree cover percentage is required')
+    .bail()
+    .isInt({ min: 0, max: 100 })
+    .withMessage('Not valid Hansen tree cover percentage 0-100')
```

Comment:
Can we add new translation file 'validations.js' and import it in 'en.js' and add these there?

Reply:
Yes, I will change this, I wanted to ask you about how to handle localized messages.

Reply:
Normally we send translation key to frontend where we use `{ t } = useTranslation()` and later in the component `t('translation.key')`
Rarely on server side we have to initialise i18n and use it there, but there are some cases (emails)

Reply:
As we are using this api only internally maybe we can keep it simple for now by no overriding the default express-validator error message, what do you think? Default error message is "Invalid value" with param name and value, something like:
{
    "value": "3",
    "msg": "Invalid value",
    "param": "opacity",
    "location": "query"
  }

Reply:
Looks good

Reply:
as msg pass a msgKey . like msgKey:'validation.errors.invalidValues'